packages/components/modules/profiles/Members/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC, Suspense } from 'react'
+
+import { LoadingState as DefaultLoadingState } from '@baseapp-frontend/design-system'
+
+import { Typography } from '@mui/material'
+import { useLazyLoadQuery } from 'react-relay'
+
+import { UserMembersListPaginationQuery as IUserMembersListPaginationQuery } from '../../../__generated__/UserMembersListPaginationQuery.graphql'
+import { UserMembersListPaginationQuery } from '../graphql/queries/UserMembersList'
+import DefaultMemberItem from './MemberItem'
+import MembersList from './MembersList'
+import { NUMBER_OF_MEMBERS_ON_FIRST_LOAD } from './constants'
+import { MemberItemSkeleton } from './styled'
+import { UserMembersProps, UserMembersSuspendedProps } from './types'
+
+const DefaultInitialLoadingState: FC = () => (
+  <>
+    {Array.from({ length: NUMBER_OF_MEMBERS_ON_FIRST_LOAD }).map((_, index) => (
+      <MemberItemSkeleton
+        key={index} // eslint-disable-line react/no-array-index-key
+        variant="rectangular"
+        sx={{ mb: 0.5 }}
+      />
+    ))}
+  </>
+)
```

Comment:
The BA packages avoid having two different FCs in the same file. I suggest you to create a component file for the loading state.